R/pkg/R/group.R

```diff
@@ -242,18 +235,73 @@ createMethods()
 setMethod("gapply",
           signature(x = "GroupedData"),
           function(x, func, schema) {
-            try(if (is.null(schema)) stop("schema cannot be NULL"))
```

Comment:
yeah, we need it. I tried to do it like dapply but dapply forces it by signature and gapply not.
will bring it back thnx